src/datasources/network/entities/network.request.entity.ts

```diff
@@ -1,4 +1,4 @@
 export interface NetworkRequest {
-  headers?: Record<string, string | number | boolean>;
+  headers?: Record<string, string>;
```

Comment:
`fetch` only permits certain header types:

```ts
type HeadersInit = [string, string][] | Record<string, string> | Headers;
```

So as to not make many changes, I adjusted this to suit the object notation.